x/slashing/handler.go

```diff
@@ -1,9 +1,30 @@
 package slashing
 
 import (
+	"strings"
+
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
+const (
+	SideChainIdBsc    int64 = 56
```

Comment:
Maybe we can initialize it with a variable `*big.Int` value instead of allocating a `big.Int` memory every time

Reply:
ok